open-machine-learning-jupyter-book/data-science/working-with-data/Basic_introduction_to_array.ipynb

```diff
@@ -0,0 +1,2529 @@
+{
```

Comment:
<p>Run all the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/615/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='160'/>